### PR TITLE
style: Collectible card

### PR DESCRIPTION
**Pull request by hachiojidev**
image

### CODE REVIEW

File: src/views/nft/components/CollectibleCard/PlaceholderImage.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
...
...DX8MbI1AgRSC8TvzoriVEWYIqwopb5nnY8AgQ4E4ndoJXEqPkwRVJQ6+PjYIgECawnE79IK4lR0mCKkKK11rzovAQIdCcTv1MLjVGyYIqAodfSxsVUCBNYWiN+tBcepyDBFOFFa+x51fgIEOhSI37GFxqm4MEUwUerw42LLBAhsJRC/awuMU1FhilCitNW96ToECHQsEL9zC4tTMWGKQKLU8cfE1gkQ2FogfvcWFKciwhRhRGnre9L1CBAgMMTv4ELilD1MEUSUfDwIECCQTSB+FxcQp6xhihCilO1mdGECBAgEgfidnDlO2cIUAUQp3BOOBAgQyC4Qv5szxilLmOLGRSn7TWgBBAgQeCgQv6MzxWnzMMUNi9LDe8HvBAgQKEYgfldniNOmYYobFaVibj4LIUCAwD6B+J29cZw2C1PcoCjtuwc8ToAAgeIE4nf3hnHaJExxY6JU3E1nQQQIEDgkEL/DN4rT6mGKGxKlQ7P3PAECBIoViN/lG8Rp1TDFjYhSsTebhREgQGCsQPxOXzlOq4UpbkCUxs7c6wgQIFC8QPxuXzFOq4QpLlyUir/JLJAAAQJTBeJ3/EpxSh6muGBRmjprrydAgEA1AvG7foU4JQ1TXKgoVXNzWSgBAgTmCsTv/MRxShamuEBRmjtj7yNAgEB1AvG7P2GckoTp+fNnw/vfjoevolTdTWXBBAgQWCqQOk6Lw3QTpXe/vh7OvnwfTk/Pl+7P+wkQIECgQoGUcVoUJlGq8O6xZAIECKwkkCpOs8MkSitN1mkJECBQsUCKOM0KkyhVfNdYOgECBFYWWBqnyWESpZUn6vQECBBoQGBJnCaFSZQauFtsgQABAhsJzI3T6DCJ0kaTdBkCBAg0JDAnTqPCJEoN3SW2QoAAgY0FpsbpYJhEaeMJuhwBAgQaFJgSpyfDJEoN3h22RIAAgUwCY+O0N0yilGlyLkuAAIGGBcbEaWeYRKnhu8LWCBAgkFkgxOnzydlw/vX7o9U8CpMoPTLyAAECBAgkFngqTvfCJEqJ5Z2OAAECBPYKvDp+Mbz95eXw8E9OMUyitNfOEwQIECCwksDxm5fD8c8v7sXprzCJ0kriTkuAAAECBwUexunof7+fXvn3lA66eQEBAgQIrChwN05Hl5eXV/6RvxW1nZoAAQIERgmEOP0fz9XbAXkyPOEAAAAASUVORK5CYII="
```
Comment (hachiojidev) [inline src/views/nft/components/CollectibleCard/PlaceholderImage.tsx:7]: Likely this is temporary.

File: src/views/nft/components/CollectibleCard/types.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { Nft } from 'config/constants/nfts/types'
+
+// Temp
+export interface Collectible {
+  name: string
+  cost: number
+  lowestCost?: number
+  nft: Nft
+  status: 'selling' | 'profile' | 'wallet'
+}
```
Comment (hachiojidev) [inline src/views/nft/components/CollectibleCard/types.ts:10]: This is also temporary until the shape is more fleshed out.

File: src/views/nft/components/CollectibleCard/PlaceholderImage.tsx (hunk 1 above)
Comment (RabbitDoge) [reply src/views/nft/components/CollectibleCard/PlaceholderImage.tsx:7]: WTF !!!

Comment (hachiojidev) [reply src/views/nft/components/CollectibleCard/PlaceholderImage.tsx:7]: Hopefully we can get an SVG but this technique is good for ensuring the placeholder image is immediately available.

File: src/views/nft/market/index.tsx [hunk 3]
```diff
@@ -1,7 +1,51 @@
 import React from 'react'
+import { Grid } from '@pancakeswap/uikit'
+import pancakeBunnies from 'config/constants/nfts/pancakeBunnies'
+import NftCard from '../components/CollectibleCard'
+import { Collectible } from '../components/CollectibleCard/types'
+
+const example1: Collectible = {
```
Comment (RabbitDoge) [inline src/views/nft/market/index.tsx:7]: maybe put all the tmp stuff in a tmp.tsx file ?

File: src/views/nft/components/CollectibleCard/index.tsx [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+`
+
+const CollectibleCard: React.FC<CollectibleCardProps> = ({ collectible, ...props }) => {
+  const { name, status, cost, lowestCost, nft } = collectible
+  const bnbBusdPrice = useBNBBusdPrice()
+
+  return (
+    <StyledCollectibleCard {...props}>
+      <CardBody p="8px">
+        <PreviewImage src={`/images/nfts/${nft.images.lg}`} height={320} width={320} mb="8px" />
```
Comment (RabbitDoge) [inline src/views/nft/components/CollectibleCard/index.tsx:32]: Tis is not going to work, we won't store nft images locally

Comment (hachiojidev) [reply src/views/nft/components/CollectibleCard/index.tsx:32]: Yeah I figured as much. Will update it once it becomes clear how the images are available.

**Review by ChefHutch — APPROVED** [review]
LGTM - just FYI @Chef-Cheems has been working on these, I'm not sure if they're an NFT Card 'variant' worth bearing in mind? In the Profile - these smaller cards are used on mobile instead of the full sized cards,

Screenshot 2021-09-07 at 08 16 21

**Pull request merged**